loopout_test.go

```diff
@@ -611,6 +643,13 @@ func testPreimagePush(t *testing.T) {
 	// Now when we report a new block and tick our expiry fee timer, and
 	// fees are acceptably low so we expect our sweep to be published.
 	blockEpochChan <- ctx.Lnd.Height + 2
+
+	err = ctx.Lnd.NotifyHeight(ctx.Lnd.Height + 2)
+	require.NoError(t, err)
+
+	err = ctx.Lnd.NotifyHeight(ctx.Lnd.Height + 2)
```

Comment:
We don't need to double notify here (and in the other tests) anymore right? (Given the mock notifier maintains a list of all subscribers).

Reply:
Correct! Probably forgot the clean-up these leftovers, as they don't alter the behavior of the batcher/batches.